server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigParser.java

```diff
@@ -433,37 +406,6 @@ private void validateGatewayConfig(GatewayConfigDocument configDoc, List<String>
     }
 
     /**
-     * Write out an input stream to a new file.
-     *
-     * @param in       the input stream
-     * @param fileName the file name
-     * @return whether file was successfully written
-     * @throws IOException
-     */
-    private void writeToNewFile(InputStream in, String fileName) throws IOException {
```

Comment:
In the 4.0 code it looked like this method was only called if an exception happened during translation, and not just in general.  I will double check but pretty sure on on error path.